src/charts/topology-map/examples/topology-map-view.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.component:pfTopologyMap
+ * @restrict E
+ *
+ * @param {array} nodes array containing objects representing graph nodes. Each node has these attributes. Only node id is mandatory parameter
+ * <ul style='list-style-type: none'>
+ *   <li>id: unique node identifier</li>
+ *   <li>title: node title</li>
+ *   <li>size: node radius; default value is 17</li>
+ *   <li>iconType: string that can be one of <b>'fonticon', 'fileicon'</b>. icon is chosen based on iconType value. If no type has been chosen, default icon will be used.</li>
+ *   <li>iconClass: if iconType is <b>fonticon</b>, node should have icon css class. eg: <b>'fa fa-info'</b></li>
+ *   <li>path: if iconType is <b>fileicon</b>, this attribute specifies path to image file. eg: <b>'/some/path/to/image/image.png'</b></li>
```

Comment:
Can't you just use `fonticon` or `fileicon` directly? The priority should be `fileicon` first...as in trees.